packages/paste-website/src/pages/experiences/artificial-intelligence/index.mdx

```diff
@@ -0,0 +1,315 @@
+- Use the [Microsoft HAX Playbook](https://microsoft.github.io/HAXPlaybook/?state=xxxxxxxx) for simulating error scenarios based on your AI scenario
+
+Remember that these methods are about building confidence, so not every one of them must be used. Use [Twilio’s generative AI design framework cards](https://www.figma.com/community/file/1392507403743352508/generative-ai-design-framework) for a list of output types, ways to give customers control, and ways to collect feedback. You can use the cards to generate different combinations of design options, and evaluate your design against other design alternatives. You can also use these cards in customer research to help customers better articulate and understand what they’d like to change about the feature.
+
+During your pilot or beta, collect customer feedback at least twice. Compared to older technologies, users’ perceptions and needs about AI will change over a shorter amount of time as they grow more familiar with the tool. Engage with your Research team so you can get more specific insights to your feature.
+
+### Content produced by AI
+
+In addition to AI features, content you produce with the help of AI should adhere to these basic guidelines:
+- If a substantial amount of your work relied on AI, clearly state that. If possible, link to the model you used.
+- Fact check the AI-produced content. LLMs produce hallucinations, resulting in content that is inconsistent with real-work facts or user inputs.
+- Make sure the AI-produced content doesn’t violate Twilio policies, especially copyright and intellectual property.
+- Verify that the model has a permissive license that allows output to be used for commercial purposes.
+
+Learn more on the [Twilio Style Guide](https://library.twilio.com/guidelines/guide/1b8c14a4-287d-47a4-8ac6-1621bfe97a6d/page/34720f98-b7f8-4436-8e66-4f814801eec1) and our “Generative AI policy”.
+
+---
+
+## Ingredients
+
+The AI UI Kit includes these components:
+- [ArtificialIntelligenceIcon](/components/icon)
+- [Badge](/components/badge)
+- [Button](/components/button)
+- [Progress Bar](/components/progress-bar)
+- [Skeleton Loader](/components/skeleton-loader)
+
+Components also used for AI chat experiences:
+- [AI Log](/components/ai-log)
+- [Avatar](/components/avatar)
+- [Chat Composer](/components/chat-composer)
+- [Side Panel](/components/side-panel)
+
+Related patterns:
+- [Error state](/patterns/error-state)
+
+<Box marginBottom="space70">
+  <Callout variant='neutral'>
+      <CalloutHeading as="h5">Need more AI components?</CalloutHeading>
+      <CalloutText>Our AI Kit was first designed primarily with chat and basic predictive and generative experiences in mind. If these components and patterns don’t meet your needs, please let us know in a [Github Discussion](https://github.com/twilio-labs/paste/discussions).</CalloutText>
```

Comment:
issue(blocking): This callout is a component. The link is not rendering in here correctly. Try using the anchor component inside